src/pages/api/revalidate.ts

```diff
@@ -26,24 +26,35 @@ async function handler(request: NextApiRequest, response: NextApiResponse) {
 	const productData = cachedGetProductData(product)
 
 	const navDataPrefixes = productData.rootDocsPaths.map(
-		({ navDataPrefix, path }) => {
-			return { navDataPrefix, path }
+		({ navDataPrefix, path, productSlugForLoader }) => {
+			return { navDataPrefix, path, productSlugForLoader }
 		}
 	)
 
 	// fetch the latest nav data, which will be used to construct paths to revalidate
-	const navDataFiles = await Promise.all(
-		navDataPrefixes.map(async ({ navDataPrefix, path }) => {
-			const prefix = navDataPrefix ?? path
-
-			const response = await fetch(
-				`https://content.hashicorp.com/api/content/${product}/nav-data/latest/${prefix}`
+	const navDataFiles = (
+		await Promise.all(
+			navDataPrefixes.map(
+				async ({ navDataPrefix, path, productSlugForLoader = product }) => {
+					const prefix = navDataPrefix ?? path
+
+					const response = await fetch(
+						`https://content.hashicorp.com/api/content/${productSlugForLoader}/nav-data/latest/${prefix}`
```

Comment:
Leverages the existing `productSlugForLoader` config value to ensure we're correctly fetching from the content API